Gemfile

```diff
@@ -54,3 +57,6 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 # Gem for rubocop
 gem "relaxed-rubocop"
```

Comment:
Needed ?
`.rubocop.yml`:
```yaml
inherit_from:
  - http://relaxed.ruby.style/rubocop.yml
(...)
```

Reply:
It's the same thing, no ? The .rubocop.yml is adapted `with inherit_gem: relaxed-rubocop: .rubocop.yml` .  I though it's better using gem when it's available.

Reply:
Principle: the fewer (dependencies) the better ... or as a fundamental of _security by design_ = always do your best trying to *minimize attack surface* ... starting with the libraries doing quite nothing !

why should we take the risk to introduce security flaws ... thanks to *files you don't control* (= could be compromised ... just FYI, it's actually happening for another -very- famous linter !) ... while it's only 2 lines to add ? (like explained in the [doc](https://github.com/janlelis/relaxed.ruby.style#usage))

But at the end of the day ... you have the _final cut_ ... since there's no big deal here, let's say it's more a question of _mindset_ ... so ... up to you !

Reply:
You understood, it was not really related to this correction (it is replaced) but generally. Thanks for the explanation !